benefits/eligibility/templates/eligibility/start__login_gov.html

```diff
@@ -0,0 +1,52 @@
+{% extends "eligibility/start.html" %}
+{% load i18n %}
+{% load core %}
+
+{% block page_title %}
+  {% translate "eligibility.pages.start.login_gov.title" %}&nbsp;|&nbsp;
+{% endblock page_title %}
+
+{% block headline %}
+  <div class="col-lg-10">
+    <h1>{% translate "eligibility.pages.start.login_gov.headline" %}</h1>
+  </div>
+{% endblock headline %}
+
+{% block media-list %}
+  {% translate "core.icons.idcardcheck" context "image alt text" as icon_alt %}
+  {% translate "eligibility.pages.start.login_gov.start_item.heading" as heading %}
+  {% translate "eligibility.pages.start.login_gov.start_item.details" as details %}
+  {# djlint: off #}
+  {% media_item icon="idcardcheck" icon_alt=icon_alt heading=heading %}
+  <div class="media-body--details">
+      <p>{{ details }}</p>
+      <div class="media-body--items">
+          <ul>
+              <li>{% translate "eligibility.pages.start.login_gov.required_items[0]" %}</li>
+              <li>{% translate "eligibility.pages.start.login_gov.required_items[1]" %}</li>
+              <li>{% translate "eligibility.pages.start.login_gov.required_items[2]" %}</li>
+          </ul>
+      </div>
+  </div>
+  {% endmedia_item %}
+  {# djlint: on #}
+  {% include "eligibility/includes/media_item__start__bankcardcheck.html" %}
+{% endblock media-list %}
+
+{% block help-link %}
+  {% url "core:help" as help_page %}
+  {# djlint:off #}
+  {% translate "eligibility.pages.start.help_text" %}
+  <a href="{{ help_page }}#login-gov">{% translate "eligibility.pages.start.help_link_text" %}</a>.
+  {# djlint:on #}
+{% endblock help-link %}
+
+{% block call-to-action-button %}
+  {% url "oauth:login" as button_url %}
+  {% translate "eligibility.buttons.senior.signin" as button_text %}
+  {# djlint:off #}
+  <a href="{{ button_url }}" class="btn btn-lg btn-primary" id="login">
+      {{ button_text }} <span class="fallback-text">Login.gov</span>
+  </a>
```

Comment:
I really like this way of being able to code buttons on their own in HTML, rather than having to mash in more logic into the view functions and `button.url` file! Gives us a lot of flexibility.